README.md

```diff
@@ -312,6 +312,13 @@ Alernative, you could pass a plain json like this:
 ```
 </details>
 
+**Send a reaction to message**
```

Comment:
nit: Could you move this under messages API?

Reply:
Sure! Also going to check the issue with it 👍